packages/internal/src/config.ts

```diff
@@ -22,6 +23,7 @@ export interface NodeTargetConfig {
 }
 
 interface BrowserTargetConfig {
+  title: string
```

Comment:
@thedavidprice and @simoncrypta This is the breaking change that I thought my be impacting e2e.

Reply:
Don't think this is a breaking change @dthyresson - there's a fallback here: https://github.com/redwoodjs/redwood/pull/3026/files/4ded4acdcd5ad39d05020168d59837879c5bc587#diff-b8b965aaf9d757f327aa2f1b7ef87083736ade531266a4d75100a8d54cd75447R152

Reply:
Like @dac09 say, It shouldn't have breaking change because if title is not set in `redwood.toml ` we use the same value we currently use for the default title in the` index.html.`
`<title><%= htmlWebpackPlugin.options.title %></title>`

The error impacting e2e is because the default title is not a string... I think I know why. I'm gonna test something!

Reply:
Its because of not using JSON.stringify as mentioned here: :) https://github.com/redwoodjs/redwood/pull/3026/files#diff-b8b965aaf9d757f327aa2f1b7ef87083736ade531266a4d75100a8d54cd75447R152

Reply:
Ahah! Thank you ! 😁